sdk/appconfiguration/Azure.Data.AppConfiguration/src/ConfigurationClient.cs

```diff
@@ -39,7 +39,7 @@ public ConfigurationClient(string connectionString)
         /// Creates a <see cref="ConfigurationClient"/> that sends requests to the configuration store.
```

Comment:
Nit, though you are consistent in your library: standard .NET text is something like `Initializes an instance of the <see cref="ConfigurationClient"/> class"` - at least as a prefix (so you could add things like "for mocking.", etc. Not saying you should change it, but it's the standard text that .NET itself uses. Key Vault and Storage, IIRC, use this form as well.